src/figure_editor.py

```diff
@@ -160,9 +160,11 @@ def __init__(self, methods, captions):
         self.update(initial_pos, initial_orient, initial_scale)
         self.setLayout(mainLayout)
 
-    def update(self, pos, orient, sacale):
-        for col, data in enumerate([pos, orient, sacale]):
+    def update(self, pos, orient, scale):
+        for col, data in enumerate([pos, orient, scale]):
             for param, val in zip(["X", "Y", "Z"], data):
+                if col == 2:
+                    val = val * 100
```

Comment:
why in general you have multiplications and divisions by 100 during scaling? to make it slower?